src/main.rs

```diff
@@ -47,6 +55,10 @@ fn main() -> failure::Fallible<()> {
         .try_init()
         .context("failed to initialize logging")?;
 
+    // Expose process start timestamp.
+    let start_time = chrono::Utc::now();
+    PROCESS_START_TIME.set(start_time.timestamp());
+
```

Comment:
This metrics belongs to the agent service. As such, this whole block would be better moved into `run_agent()`, right after the `info!()` logging.

Reply:
Thanks for review. Good point. I will move to after the `starting update agent` info.